README.md

```diff
@@ -1,46 +1,163 @@
 # Catalyst
 
+The entire bundle of application needed for booting up livepeer's
+decentralised media server in place is termed as [catalyst].
+
 ## Getting Started
 
-go1.17 is recommended because building with < 1.17 (specifically 1.16) can result in issue on arm64 architectures.
+Make sure you have the following installed on the machine before
+proceeding:
 
-From there:
+  - go1.17± or newer
+  - ffmpeg v5 (or newer)
+  - GNU make (not required, but good to have for using `Makefile`
+    commands)
+  - docker version 19 (or newer) (not mandatory for basic catalyst
```

Comment:
**D**ocker Version 19 or newer (not mandatory for local developement)

Reply:
:+1: